apps/rehabstod/src/components/Table/utils/filterTableColumns.ts

```diff
@@ -6,11 +6,18 @@ export const filterTableColumns = (
   showPersonalInformation?: boolean,
   onlyVisible?: boolean,
   srsActivated?: boolean,
-  excludedColumns?: string[]
+  excludedColumns?: string[],
+  isPatientView?: boolean
 ) =>
   columns
     .filter(({ visible }) => visible || !onlyVisible)
     .filter(({ name }) => !(showPersonalInformation === false && (name === 'Namn' || name === 'Personnummer')))
     .filter(({ name }) => !(isDoctor && name === 'Läkare'))
     .filter(({ name }) => !(!srsActivated && name === 'Risk'))
+    .filter(({ name }) => !(isPatientView && (name === 'Namn' || name === 'Personnummer' || name === 'Kön' || name === 'Ålder')))
```

Comment:
I think this filter has grown alot. Instead of building on top of it, I think it would be a good idea to create a separate filter function, like we already one for filterHiddenColumns for example.

This will increase the readbility in my opinion and makes the code easier to understand.

Reply:
Good idea! I'll fix this and the comment below in the code I've already written for AG to avoid merge conflicts.